cmd/restore.go

```diff
@@ -27,5 +27,5 @@ func Restore(ctx *cli.Context) {
 	if err := hw.Save(); err != nil {
 		log.Fatal(err)
 	}
-	hw.Summarize()
+	hw.Summarize(false)
```

Comment:
I was thinking of making a struct to take options on what to summarize but decided it wasn't worth the complexity if we didn't need the functionality.

Can you give me an example of what you mean? I definitely do agree that it could be more readable.